Explorer/Assets/DCL/SDKComponents/AudioSources/Systems/UpdateAudioSourceSystem.cs

```diff
@@ -62,8 +62,15 @@ private void CreateAudioSource(ref PBAudioSource sdkAudioSource, ref AudioSource
                 return;
 
             if (!audioSourceComponent.AudioSourceAssigned)
-                audioSourceComponent.SetAudioSource(audioSourcesPool.Get(), audioMixerGroup);
+            {
+                if (audioSourcesPool == null)
+                {
+                    var a = 1;
```

Comment:
you left debug code?

Reply:
I didnt merge a file xDDDDD

Reply:
And it was the file with the specific changes that fixed everything 🤦